client/common/cmdline.c

```diff
@@ -2527,6 +2528,17 @@ int freerdp_client_settings_parse_command_line_arguments(rdpSettings* settings,
 			if (!(settings->ActionScript = _strdup(arg->Value)))
 				return COMMAND_LINE_ERROR_MEMORY;
 		}
+		/* Enable FIPS Mode, which forces the following and overrides the following(by happening later */
+		/* in the command line processing): */
+		/* 1. Disables NLA Security since NLA in freerdp uses NTLM(no Kerberos support yet) which uses algorithms */
+		/*	not allowed in FIPS for sensitive data. So, we disallow NLA when FIPS is required. */
+		/* 2. Forces the only supported RDP encryption method to be FIPS. */
+		CommandLineSwitchCase(arg, "fipsmode")
+		{
+			settings->FIPSMode = TRUE;
+			settings->NlaSecurity = FALSE;
```

Comment:
I agree, I had it in my head that putting this at the end of the list was solving that, but looking back that is clearly not the case.